apps/system/js/nfc_handover_manager.js

```diff
@@ -324,21 +311,20 @@
      */
     _doPairing: function _doPairing(mac) {
       this._debug('doPairing: ' + mac);
-      if (this.defaultAdapter == null) {
-        // No BT
-        this._debug('No defaultAdapter');
+      var adapter = Service.query('Bluetooth.getAdapter');
+      if (adapter === null) {
+        this._debug('No Bluetooth Adapter');
         return;
       }
-      var req = this.defaultAdapter.pair(mac);
-      var self = this;
-      req.onsuccess = function() {
-        self._debug('Pairing succeeded');
-        self._clearBluetoothStatus();
-        self._doConnect(mac);
+      var req = adapter.pair(mac);
+      req.onsuccess = () => {
```

Comment:
Be careful that Bluetooth v2 API `adapter.pair()` is a `Promise` method. I'm wondering the old Dom request coding style might not work on v2. Please check the functionality on v2. Thanks.

https://wiki.mozilla.org/B2G/Bluetooth/WebBluetooth-v2/BluetoothAdapter#pair.28DOMString_aDeviceAddress.29